docs/section5/Rule5-11.md

```diff
@@ -0,0 +1,31 @@
+
+# Envelope - Rule 5-11  
+
+**Rule ID:** 5-11  
+**Rule Description:**  Manual window shading devices such as blinds or shades are not required to be modeled.  
+**Rule Assertion:** Baseline RMR = expected value  
+**Appendix G Section:** Section G3.1-5(d) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+
+  - For each zone in thermal block: ```for zone_baseline in thermal_block_baseline.zones:```  
+
+  - For each space in thermal zone: ```for space_baseline in zone_baseline.spaces:```  
+
+    - Get surface in space: ```for surface_baseline in space_baseline.surfaces:```  
+
+      - Get vertical glazings in exterior wall: ```if ( surface_baseline.classification == "WALL" ) AND ( surface_baseline.adjacent_to == "AMBIENT" ): windows_baseline = surface_baseline.fenestration_subsurfaces```  
+
+      **Rule Assertion:** Baseline vertical fenestrations are not modeled with manual window shading devices: ```window.has_manual_interior_shades == FALSE for window in windows_baseline```  
```

Comment:
This is a weird requirement since Appendix G says something "is not required to be modeled" so I think maybe we should issues warning when they are modeled. 

@mkarpman what do you think?

Reply:
@JasonGlazer I agree - similar to how we would issue a warning if daylighting is modeled in the baseline, we should issue a warning if manual internal shades are modeled in the baseline.

Reply:
I think modeling daylighting in the baseline should trigger an error, because I don't think there are any circumstances when this would be correct. For manual shading, the rules is that it should be modeled the same in the baseline and proposed. That should be rule assertion.

Reply:
This does not seem to be resolved. I think this should check that manual fenestration in baseline must be the same as in proposed. So perhaps rule assertion needs to be updated? Sorry if I am missing something here.